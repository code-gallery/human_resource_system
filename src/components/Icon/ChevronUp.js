import React from 'react'
import PropTypes from 'prop-types'

const ChevronUp = ({ color, className }) => (
  <svg
    className={className}
    viewBox="0 0 18 12"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g fill="none" fillRule="evenodd">
      <g transform="translate(-1326 -576)" stroke={color} strokeWidth="3">
        <polyline
          id="Path-3"
          transform="rotate(-180 1335.486 582.995)"
          points="1328 579 1335.7685 586.99 1342.97118 579"
        />
      </g>
    </g>
  </svg>
)

ChevronUp.propTypes = {
  color: PropTypes.string.isRequired,
  className: PropTypes.string
}

ChevronUp.defaultProps = {
  className: ''
}

export default ChevronUp
