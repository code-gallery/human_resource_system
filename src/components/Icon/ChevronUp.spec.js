import React from 'react'
import renderer from 'react-test-renderer'
import ChevronUp from './ChevronUp'

describe('ChevronUp Icon', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <ChevronUp color="#000" />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
