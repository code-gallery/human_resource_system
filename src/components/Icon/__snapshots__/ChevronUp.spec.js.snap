// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChevronUp Icon renders correctly 1`] = `
<svg
  className=""
  viewBox="0 0 18 12"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    fill="none"
    fillRule="evenodd"
  >
    <g
      stroke="#000"
      strokeWidth="3"
      transform="translate(-1326 -576)"
    >
      <polyline
        id="Path-3"
        points="1328 579 1335.7685 586.99 1342.97118 579"
        transform="rotate(-180 1335.486 582.995)"
      />
    </g>
  </g>
</svg>
`;
